src/trio/_timeouts.py

```diff
@@ -7,36 +7,42 @@
 import trio
 
 
-def move_on_at(deadline: float) -> trio.CancelScope:
+def move_on_at(deadline: float, *, shield: bool = False) -> trio.CancelScope:
     """Use as a context manager to create a cancel scope with the given
     absolute deadline.
 
     Args:
       deadline (float): The deadline.
+      shield (bool): Initial value for the `~trio.CancelScope.shield` attribute
+          of the newly created `cancel scope
+          <https://trio.readthedocs.io/en/stable/reference-core.html#trio.CancelScope.shield>`__.
```

Comment:
Shouldn't sphinx be able to detect `CancelScope` reference if you changed text to `~trio.CancelScope` or something? I'm pretty sure there's a way to do that, but I don't remember.